cobbler/cobbler_collections/manager.py

```diff
@@ -25,18 +25,29 @@
 import weakref
 import uuid
 
+from cobbler.api import CobblerAPI
```

Comment:
Has to be removed due to circular import.

Reply:
I have decided to purge all Cobbler Types from the typing due to the fact that we can't prohibit these circular imports due to the modular architecture of Cobbler. This will increase the size of this PR but well it doesn't help.

Reply:
Sounds reasonable.